packages/components/src/server-side-render/index.js

```diff
@@ -86,7 +86,7 @@ export class ServerSideRender extends Component {
 	render() {
 		const response = this.state.response;
 		const { className } = this.props;
-		if ( ! response ) {
+		if ( ! response && response !== '' ) {
```

Comment:
This check is confusing. Is it possible to get a `response` which isn't an object or `null`? If `response` would be a non-empty string then condition `response.error` and `response.length` will work quite randomly.

Reply:
Hi @gziolo, I updated the flow of logical conditions to make the code easier to understand.